### PR TITLE
Fine-Tuning Scheduler Tutorial Update for Lightning/PyTorch 2.2.0

### PR DESCRIPTION
**Pull request by speediedan**
Validated the Fine-Tuning Scheduler (FTS) tutorial for FTS/Lightning/PyTorch `2.2.0` (as of the final PyTorch RC and the [recent `2.2.0` Lightning commit](https://github.com/Lightning-AI/lightning/tree/c989a97aa196431ecb70e1a0f8c839cdeaba9300)). 

The only minor change in this PR is to remove references/links to `tensorboard.dev` since that service is shutting down on 2023.12.31. 

## Before submitting

- [X] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [X] Did you make sure to **update the docs**?
- [X] Did you write any new **necessary tests**?

## What does this PR do?

Updates the Fine-Tuning Scheduler tutorial to prepare for `2.2.0` tutorial publishing.

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by speediedan** [comment]
Note only error is a [link resolution error](https://github.com/Lightning-AI/tutorials/actions/runs/7341685319/job/19989776647?pr=298#step:14:494) unrelated to this PR or tutorial.

**Comment by Borda** [comment]
@speediedan apology for the long delay but finally got the repo work again...

**Review by Borda — APPROVED** [review]

**Comment by speediedan** [comment]
> @speediedan apology for the long delay but finally got the repo work again...

@Borda Thanks for streamlining the tutorials publishing process and fixing the outstanding issues! Love the simplification dropping the largely duplicative `nbval` part of the flow. Thanks again for all your work!
